src/components/WalletAuthModal.tsx

```diff
@@ -116,20 +114,8 @@ export const WalletAuthModal = ({ open }: { open: boolean }) => {
         web3Context.deactivate();
       }
     }
-
-    setConnectMessage('');
   };
 
-  useEffect(() => {
-    if (
-      // safe to refer to window here because we are in useEffect -g
-      window.location.search === AUTO_OPEN_WALLET_DIALOG_PARAMS ||
-      walletAuth.connectorName
-    ) {
-      walletAuth.connectorName && activate(walletAuth.connectorName);
```

Comment:
if i'm not mistaken, this is what allows the app to automatically reconnect to the last logged-in wallet, so it shouldn't be removed.

we're thinking alike though -- i took the auto-open stuff out in my branch here: d338a48

Reply:
ok, yeah, you're right. just looked into this more and on page reload, or fresh visit, the walletAuth.connectorName is present as `injected` but the walletAuth is not activated. so we need this, otherwise you have to click "metamask" every page reload.